pom.xml

```diff
@@ -77,8 +77,6 @@ http://www.w3.org/2001/XMLSchema-instance">
         <authentication.class.name>com.intuit.wasabi.authentication.impl.DefaultAuthentication
         </authentication.class.name>
 
-        <assign.user.to.old>true</assign.user.to.old>
-        <assign.user.to.new>false</assign.user.to.new>
```

Comment:
with the removal of these 2 props, it will be silently switched to the new table once we deploy this new code. since we are not writing to the old table anymore, there will be no way to roll back to previous code if things go wrong.